### PR TITLE
initial commit exercise1 problems1-12

### PR DESCRIPTION
**Pull request by ssakyp**
Completed exercise1 problems

### CODE REVIEW

**Comment by talgat-ruby** [comment]
@ssakyp  push to dedicated commit

File: exercise1/problem4/problem4_test.go
Comment (talgat-ruby) [inline exercise1/problem4/problem4_test.go:1]: Pull changes from my repo. Don't update tests!

**Review by talgat-ruby — CHANGES_REQUESTED** [review]

File: exercise1/problem4/problem4_test.go
Comment (ssakyp) [reply exercise1/problem4/problem4_test.go:1]: Noted. What should I do now? Close this pull request "initial commit exercise1 problems1-12 #5"? Cause I have the second pull request.

Comment (talgat-ruby) [reply exercise1/problem4/problem4_test.go:1]: yeah sure, please close it!

**Pull request closed without merging**